SmartTutor/ContentModel/LearningObjects/LearningObject.cs

```diff
@@ -1,12 +1,16 @@
-﻿namespace SmartTutor.ContentModel.LearningObjects
+﻿using System.Collections.Generic;
+using SmartTutor.ProgressModel.Progress;
+
+namespace SmartTutor.ContentModel.LearningObjects
 {
     public class LearningObject
     {
         public int Id { get; private set; }
         public int LearningObjectSummaryId { get; private set; }
+        public List<NodeProgress> NodeProgresses { get; set; }
```

Comment:
Can we remove this association?

There is no use case where we will want to access all the node progresses that contain a particular LO. Furthermore, the content model should be minimally aware of the progress model, and this adds a dependency that we don't need.

You can setup at the EF Code configuration level (SmartTutorContext) the many-to-many relationship.

Reply:
Yes, I agree. I've used this to fix the issue where the Learning Object was being connected to only one Node Progress, which was causing issues when a knowledge node is loaded multiple times (and also wasn't consistent with the intended model). I'll configure the right relationship in the the SmartTutorContext 😄